public-site/src/pages/heroines.js

```diff
@@ -55,15 +70,12 @@ const HeroinesPage = ({ data }) => (
         <Card color={node.frontmatter.color} key={node.id}>
           <Link to={node.fields.slug}>
             <h2>{node.frontmatter.heroine}</h2>
+            <p class="smol">{node.frontmatter.title}</p>
             <Img
               fluid={node.frontmatter.thumbnail.childImageSharp.fluid}
               alt={`${node.frontmatter.heroine}'s Thumbnail Illustration`}
             />
-            <p>Title: {node.frontmatter.title}</p>
             <p>{node.excerpt}</p>
-            <CardColor color={node.frontmatter.color}>
```

Comment:
We probably should open an issue to remind ourselves to add this to the detail page somewhere.